public/widget/discrepancy-note/Dn.js

```diff
@@ -617,7 +617,7 @@ Comment.prototype._getRows = function( item ) {
     msg = item.comment || item.message;
     elapsed = this._getParsedElapsedTime( item.date_time );
     fullName = this._parseFullName( item.user ) || me;
-    return '<tr><td>' + ( types[ item.type ] || '' ) + '</td><td>' + msg + '</td><td>' + fullName + '</td><td>' + elapsed + '</td></tr>';
+    return '<tr><td colspan="2">' + ( types[ item.type ] || '' ) + msg + '</td><td>' + fullName + '</td><td>' + elapsed + '</td></tr>';
```

Comment:
We only have 3 columns remaining right? I'm trying to figure out if colspan="2" is necessary here.